Calculate.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Numerics;
+
+
+namespace Calculate.Tests;
+
+[TestClass]
+public sealed class ProgramTests
+{
+    [TestMethod]
+    public void Program_Constructor_DefinesNonNullDelegates()
```

Comment:
nit: I'm not sure how useful this test is and could probably be removed. I think the other tests will reveal if the program defines the delegates correctly just by using them.